client/scss/components/_panel.scss

```diff
@@ -160,3 +160,21 @@ $header-button-size: theme('spacing.6');
 .w-panel__wrapper {
   @include max-form-width();
 }
+
+.w-panel--dashboard {
+  @include box();
+  padding-bottom: 0;
+  margin-bottom: calc(
+    theme('spacing.4') + theme('spacing.4') * var(--w-density-factor)
+  );
+
+  .w-panel__header {
+    margin-inline-start: calc(-1 * theme('spacing.5'));
+    padding-bottom: theme('spacing.5');
+  }
+
+  .w-panel__heading {
+    margin-inline-start: theme('spacing.2');
+    white-space: wrap;
```

Comment:
Should we be adding this wrapping for all panels, not just the dashboard ones?

Reply:
Probably, yes. There's an intentional [`white-space: nowrap;`](https://github.com/wagtail/wagtail/blob/93b9c04ce75ad8581852ef1d669ce43455e83454/client/scss/components/_panel.scss#L43) set for headings, and I'm hesitant about overriding it globally without testing panels on other pages.